scripts/Muon/GUI/Common/fitting_tab_widget/fitting_tab_presenter.py

```diff
@@ -607,6 +608,12 @@ def get_selected_fit_workspaces(self):
         else:
             return []
 
+    def instrument_changed(self):
+        self.deactivate_tf_asymmetry()
+
+    def deactivate_tf_asymmetry(self):
```

Comment:
```suggestion
```

Reply:
Are you recommending removing deactivate_tf_asymmetry and put it's code into instrument_changed? My thinking was in case anything else needs to be done inside instrument changed it can be done as a series of method calls

Reply:
its a simple call, so it not really worth putting into its own function